engine/unity5/Assets/Scripts/GUI/MainMenu.cs

```diff
@@ -66,25 +66,50 @@ public bool MyRemoteCertificateValidationCallback(System.Object sender, X509Cert
 
         private void Awake()
         {
-            WebClient client = new WebClient();
-            ServicePointManager.ServerCertificateValidationCallback = MyRemoteCertificateValidationCallback;
-            var json = new WebClient().DownloadString("https://raw.githubusercontent.com/Autodesk/synthesis/master/VersionManager.json");
-            VersionManager update = JsonConvert.DeserializeObject<VersionManager>(json);
-            updater = update.URL;
-
             string CurrentVersion = "4.2.0.1";
-            Auxiliary.FindObject(gameObject, "ReleaseNumber").GetComponent<Text>().text = "Version " + CurrentVersion;
 
-            var localVersion = new Version(CurrentVersion);
-            var globalVersion = new Version(update.Version);
+            if (CheckConnection())
+            {
+                WebClient client = new WebClient();
+                ServicePointManager.ServerCertificateValidationCallback = MyRemoteCertificateValidationCallback;
```

Comment:
This Remote Certificate Validation Callback is what allows us to make checks on the server over Unity. So this will need to  be added to the `CheckConnection()` method as well in order for us to make that callback.